### PR TITLE
Auto-generation of camera plugin

### PR DESCRIPTION
**Pull request by julianoes**
This adds the auto-generation of the camera plugin.

### CODE REVIEW

File: src/backend/src/plugins/telemetry/telemetry_service_impl.h [hunk 1]
```diff
@@ -1115,7 +1115,7 @@ class TelemetryServiceImpl final : public rpc::telemetry::TelemetryService::Serv
 
         _telemetry.home_async(
             [this, &writer, &stream_closed_promise, is_finished, &subscribe_mutex](
-                const mavsdk::Telemetry::Position& home) {
+                const mavsdk::Telemetry::Position home) {
```
Comment (JonasVautherin) [inline src/backend/src/plugins/telemetry/telemetry_service_impl.h:1118]: Do we want to remove that `&`, or is that a mistake in the template?

Comment (julianoes) [reply src/backend/src/plugins/telemetry/telemetry_service_impl.h:1118]: Hm, this probably happened by mistake. In general I would move to pass by value / copy as much as possible and then profile and optimize if needed.

**Comment by julianoes** [comment]
The CI error is the usual dead lock on destruction as addressed in #1011.

File: src/backend/src/plugins/telemetry/telemetry_service_impl.h (hunk 1 above)
Comment (JonasVautherin) [reply src/backend/src/plugins/telemetry/telemetry_service_impl.h:1118]: Ok

**Comment by JonasVautherin** [comment]
Haha not waiting for the iOS test? :sweat_smile:

**Pull request merged**